### DISCUSSION TITLE
php_network_getaddresses: getaddrinfo failed

### DISCUSSION THREAD

**Discussion by ghost** [pimcore/pimcore#12023] [Q&A / Support]
Hi,

I am facing this error while implementing Redis with pimcore.
![image](https://user-images.githubusercontent.com/52573912/166476683-99817c94-03ba-453a-9d09-5a7dc1c831c4.png)

Configuration on # config/packages/cache.yaml
![image](https://user-images.githubusercontent.com/52573912/166475119-694e5303-2258-4cf5-ab22-606aac8e9fc8.png)
env variable set like:
![image](https://user-images.githubusercontent.com/52573912/166476989-9fead59a-2411-40b8-b642-fe0d4d385289.png)

I have installed Redis on my system. and this is working something like this.
![image](https://user-images.githubusercontent.com/52573912/166475528-7d456a71-f9ed-4d65-8b36-bb115ea79b5c.png)

also Redis-cli working on 127.0.0.1:6379
![image](https://user-images.githubusercontent.com/52573912/166476319-01a3c792-9b96-4593-a494-8966b3030daf.png)

also If I execute **bin/console pimcore:system:requirements:check** this command and it's return status OK for redis.
![image](https://user-images.githubusercontent.com/52573912/166475894-6c955b2e-52da-433b-979b-2185a9fc8eb3.png)

I can't get why pimcore admin is throwing this error.

Pls help if anyone know about this error.

**Comment by kingjia90** [comment]
not sure, but the problem could be that the REDIS_HOST env variable is `redis:127.0.0.1`, the correct format should be like `redis://127.0.0.1`

**Reply by ghost** [reply]
@kingjia90  I have tried this also, But still does not work.

**Reply by kingjia90** [reply]
Ok, then i would retry with `redis://localhost:6379` (it's not using the port env anywhere) and remember to clear cache by `bin/console cache:clear`, hope it helps.

**Reply by ghost** [reply]
It's working but with error, like on admin load we get the error
![image](https://user-images.githubusercontent.com/52573912/166884387-5c2f9468-1738-45b7-9cc2-03a13254d26e.png)

**Reply by kingjia90** [reply]
is `redis.conf` properly set up? https://pimcore.com/docs/pimcore/current/Development_Documentation/Development_Tools_and_Details/Cache/index.html#page_Recommended-Redis-Configuration
This `maxmemory-policy volatile-lru` is "required" to work properly, and it seems related to the [eviction policy](https://docs.redis.com/latest/rs/databases/configure/eviction-policy/) in the screenshot
